environments/icds-cas/app-processes.yml

```diff
@@ -0,0 +1,283 @@
+  'pillow0':
+    kafka-ucr-static-cases:
+      start_process: 0
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 0
+      num_processes: 4
+      total_processes: 36
+  'pillow1':
+    kafka-ucr-static-cases:
+      start_process: 6
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 4
+      num_processes: 4
+      total_processes: 36
+  'pillow2':
+    kafka-ucr-static-cases:
+      start_process: 12
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 8
+      num_processes: 4
+      total_processes: 36
+  'pillow3':
+    kafka-ucr-static-cases:
+      start_process: 18
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 12
+      num_processes: 4
+      total_processes: 36
+  'pillow4':
+    kafka-ucr-static-cases:
+      start_process: 24
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 16
+      num_processes: 4
+      total_processes: 36
+  'pillow5':
+    kafka-ucr-static-cases:
+      start_process: 30
+      num_processes: 6
+      total_processes: 36
+    CaseToElasticsearchPillow:
+      start_process: 20
+      num_processes: 4
+      total_processes: 36
+  'pillow10':
+    CaseToElasticsearchPillow:
+      start_process: 24
+      num_processes: 6
+      total_processes: 36
+  'pillow11':
+    CaseToElasticsearchPillow:
+      start_process: 30
+      num_processes: 6
+      total_processes: 36
+  'pillow6':
+    kafka-ucr-main:
+      num_processes: 1
+    KafkaDomainPillow:
+      num_processes: 1
+    LedgerToElasticsearchPillow:
+      num_processes: 1
+    SqlSMSPillow:
+      num_processes: 1
+    XFormToElasticsearchPillow:
+      num_processes: 4
+    kafka-ucr-static-forms:
+      start_process: 8
+      num_processes: 4
+      total_processes: 16
+  'pillow7':
+    FormSubmissionMetadataTrackerPillow:
+      num_processes: 8
+    kafka-ucr-static-forms:
+      start_process: 12
+      num_processes: 4
+      total_processes: 16
+  'pillow8':
+    kafka-ucr-static-forms:
+      start_process: 0
+      num_processes: 8
+      total_processes: 16
+  'pillow9':
+    AppDbChangeFeedPillow:
+      num_processes: 1
+    ApplicationToElasticsearchPillow:
+      num_processes: 1
+    CacheInvalidatePillow:
+      num_processes: 1
+    UserGroupsDbKafkaPillow:
+      num_processes: 1
+    UserPillow:
+      num_processes: 1
+    DefaultChangeFeedPillow:
+      num_processes: 1
+    DomainDbKafkaPillow:
+      num_processes: 1
+    GroupPillow:
+      num_processes: 1
+    GroupToUserPillow:
+      num_processes: 1
+    kafka-ucr-static-awc-location:
+      num_processes: 1
+    UpdateUserSyncHistoryPillow:
+      num_processes: 1
+    UserCacheInvalidatePillow:
+      num_processes: 1
+  'pillow10':
+  'pillow11':
+  'pillow12':
+  'pillow13':
+  'pillow14':
```

Comment:
We'll probably need to add some partitions to kafka for both cases and forms. Should add this to the migration doc?